packages/assistify-help-request/server/migrations.js

```diff
@@ -13,19 +13,38 @@ const removeObsoleteSubscriptions = function() {
 				'localField': 'u._id',
 				'foreignField': '_id',
 				'as': 'users'
-			}},
-		{ '$match': { 'users._id': { '$exists': false } } }
+			}
+		},
+		{'$match': {'users._id': {'$exists': false}}}
 	]).forEach(function(errSubscription) {
 		// we can bypass the cached collection as this will affect obsolete values
-		RocketChat.models.Subscriptions.model.remove({ _id: errSubscription._id.toString() });
+		RocketChat.models.Subscriptions.model.remove({_id: errSubscription._id.toString()});
 		counter++;
 	});
 	return counter;
 };
 
+function makePrivateRoomsSecret() {
+	let count = 0;
+	RocketChat.models.Rooms.findByType('p').forEach((room) => {
+		if ('customFields' in room) {
+			if ('secretRoom' in room.customFields) {
+				return;
+			}
+		}
+		RocketChat.models.Rooms.setPrivacyById(room._id, true);
```

Comment:
In the migration, you don't necessarily need to use a method. You can set it in one DB statement: `update({t: 'p', hidden: {$exists: false}}, {$set: {hidden: true}}, {multi: true})`

Reply:
Handled using the Base model `update` method.